### ISSUE TITLE
MA Lead Moodle Books

### ISSUE THREAD

**Issue by kmarjanovic** [TWUOnline/bug-tracking#894]
### Instructions
- [ ] Please see the following schedule for MA Lead Courses.  All of them need to have Moodle Books.  
- [ ] Please use [MA Lead Course Template (6 weeks)](https://learn.twu.ca/course/view.php?id=30164&section=0#tabs-tree-start) as the template - or [LDRS 502 6 weeks](https://learn.twu.ca/course/view.php?id=28254&section=0#tabs-tree-start)

**September Start**

- [x] [**LDRS 500** Leadership Foundations](https://learn.twu.ca/course/view.php?id=28198&section=0#tabs-tree-start)  - done
- [ ] [**LDRS 501** Strategic Leadership](https://learn.twu.ca/course/view.php?id=28347&section=0#tabs-tree-start) ***TOP PRIORITY for next week (July 18?)
- [x] [**LDRS 502** Team Leadership and Conflict Resolution](https://learn.twu.ca/course/view.php?id=28254&section=0#tabs-tree-start) (S) – done (this is the template to use) 
- [x] [LDRS 504 Leadership Values and Ethics](https://learn.twu.ca/course/view.php?id=28349&section=0#tabs-tree-start) - done (Kelly to check assessment)

- [ ] Please create Moodle Shells and Moodle books for the following:  (Note that no units are deleted, so all should be in Books)

- [x] **LDRS 612** Leading Change and Innovation 
- [x] **LDRS 617** Employee Engagement and Stewardship
- [x] LDRS 620 Worldview Foundations of Educational Leadership 
- [ ] LDRS 625 Educational Leadership and Change  
- [x] LDRS 626 Leadership for Contemporary Issues in Education  (no master; SME will create)
- [ ] LDRS 631 Health Leadership Issues 
- [x] LDRS 632 Leadership and Change – Healthcare stream 

**October Start**

- [x] **LDRS 614** Mission, Marketing, and Quality 
- [x] LDRS 623 Developing and Assessing Educational Programs 
- [x] LDRS 634 Accountability and Performance

**Comment by meher-ahmed** [comment]
Noted @kmarjanovic

**Comment by kmarjanovic** [comment]
@cmadland @Jeff-TWU @meher-ahmed @MeeksonHundoo 
Just an update:  
- Please use [MA Lead Course Template (6 weeks)](https://learn.twu.ca/course/view.php?id=30164&section=0#tabs-tree-start) as a template for all MA Lead courses.
- There should be just 8 tabs (home, Weeks 1-6, & Assessment
- @Jeff-TWU do we need a [Course Resources tab](https://learn.twu.ca/course/view.php?id=30164&section=8#tabs-tree-start) for the files, or can we just embed the links in the Moodle books (preferred method)?

All Sept courses are due August 15.  If possible, we need LDRS 501 this week.

For the process:
- please let me know when a course is ready in Moodle so I can review
- I'll let DeAnn know to notify instructors that the course is ready to import.

**Comment by Jeff-TWU** [comment]
> @cmadland @Jeff-TWU @meher-ahmed @MeeksonHundoo Just an update:
> 
> * Please use [MA Lead Course Template (6 weeks)](https://learn.twu.ca/course/view.php?id=30164&section=0#tabs-tree-start) as a template for all MA Lead courses.
> * There should be just 8 tabs (home, Weeks 1-6, & Assessment
> * @Jeff-TWU do we need a [Course Resources tab](https://learn.twu.ca/course/view.php?id=30164&section=8#tabs-tree-start) for the files, or can we just embed the links in the Moodle books (preferred method)?
@kmarjanovic The files have to live somewhere on Moodle. They should be referenced in the different unit books but they will always need to live somewhere. For this reason I recommend having a course resources tab. This tab can be hidden from students.
> 
> All Sept courses are due August 15. If possible, we need LDRS 501 this week.
> 
> For the process:
> 
> * please let me know when a course is ready in Moodle so I can review
> * I'll let DeAnn know to notify instructors that the course is ready to import.

**Comment by kmarjanovic** [comment]
- [ ] Could you please copy all of the [MA Lead Template](https://learn.twu.ca/course/view.php?id=30164&section=0#tabs-tree-start) content to [LDRS 502: Team Ldrship & Conflict Resolution (6 week MASTER - 2023)](https://learn.twu.ca/course/view.php?id=28254&section=0#tabs-tree-start).

- [ ] The course intro of 502 shouldn't be replaced, but all of the other tabs need the Master Template content.  

Please let me know when it's done so I can change the intro to the Weeks.

This will be the intro blurb - but it for 502, for example, each week will have 1 unit (most other MA Lead courses will have 10 units...Week 1: Intro & Unit 1; Wk2: Unit 2&3; Wk3: Unit 4&5; Wk4: Unit 6&7; Wk5: Unit 8&9; Wk6: Unit 10

Welcome to Week # of LDRS #!
On your own: Work through the content and learning activities in Units # & #.
Meet with your class. Be sure to check the [announcements](https://learn.twu.ca/mod/forum/view.php?id=915814) for specific required activities and resources.
See below for the course materials you will complete this week, which include learning outcomes, learning activities, and resources. Be sure to review the assessment instructions, including grading criteria, and submit your assignments by the assigned due date.

**Comment by kmarjanovic** [comment]
@cmadland @Jeff-TWU @meher-ahmed @MeeksonHundoo 

DeAnn and I went through the courses this morning, and there are a number of items to fix & complete.

For all MA Lead courses, please use the updated [MA Lead Course Template](https://learn.twu.ca/course/view.php?id=30164&section=0#tabs-tree-start).  See the comment above for the changes.

Today I'll fix 500, 501, 502, and 504 (needed for tomorrow).

Could someone work on the other courses this week?  LDRS 617, 631, 632 and 697 are top priority, as the new instructors need to see them this week.

The other courses needed for September are: LDRS 612, 620, and 625.

@deannadams I forgot to ask if the SME (?) for LDRS 626 will need help setting up her course?  You can import the MA Lead Master for the instructor to work on. (we haven't developed this course).

Thanks all!!

**Comment by kmarjanovic** [comment]
LDRS 500, 501, & 502 are good to go!

For [LDRS 504 (6 week MASTER - 2023)](https://learn.twu.ca/course/view.php?id=28349&section=8)

Please add the assessment dropboxes for the assignments & discussions.
see bookdown: https://ma-lead.github.io/ldrs504/meeting-the-ethical-challenge-of-leadership.html#assessment

See the MA Lead template for formatting: https://learn.twu.ca/course/view.php?id=30164&section=7#tabs-tree-start

Thank you!

**Comment by deannadams** [comment]
@kmarjanovic - I don't even think we have identified a SME for LDRS 626. Will add this to my list to talk about with Mark (hopefully tomorrow at 11).

**Comment by kmarjanovic** [comment]
Good morning all!
@MeeksonHundoo or @meher-ahmed - could you please check the gradebooks for the completed MA Lead courses?  Here are the priority courses: 500, 501, 502, **504**, 617, 631, 632, and 697.

Please let @deannadams know when the LDRS 504 gradebook is complete:
504 assessment:
Discussion forums – 25% (5% each)
Individual Assignment Papers – 20% (10% each)
Gibbs Analysis Reflection Paper – 30% (15% each)
Final Assignment – 25%

**Comment by MeeksonHundoo** [comment]
> Good morning all! @MeeksonHundoo or @meher-ahmed - could you please check the gradebooks for the completed MA Lead courses? Here are the priority courses: 500, 501, 502, **504**, 617, 631, 632, and 697.
> 
> Please let @deannadams know when the LDRS 504 gradebook is complete: 504 assessment: Discussion forums – 25% (5% each) Individual Assignment Papers – 20% (10% each) Gibbs Analysis Reflection Paper – 30% (15% each) Final Assignment – 25%

Good morning. Hello @deannadams 504 gradebook is done. Here is the link to it:
https://learn.twu.ca/grade/edit/tree/index.php?id=28349

**Comment by MeeksonHundoo** [comment]
Hello @deannadams ldrs500 gradebook is done. Here is the link: 
https://learn.twu.ca/grade/edit/tree/index.php?id=28198
If you could help me source the syllabus for the remainder of the courses to setup their respective gradebook's that would be great.

**Comment by kmarjanovic** [comment]
Hi all,
Just confirmed with Colin text that we can follow the template for the week intro.  
@MeeksonHundoo & @meher-ahmed could you check the Moodle Books have the Overview & Learning Outcomes.  Please take it out of the week tab info and move back to the books.

Intro blurb:
Welcome to Week # of LDRS #!
On your own: Work through the content and learning activities in Units # & #.
Meet with your class. Be sure to check the [announcements](https://learn.twu.ca/mod/forum/view.php?id=915814) for specific required activities and resources.
See below for the course materials you will complete this week, which include learning outcomes, learning activities, and resources. Be sure to review the assessment instructions, including grading criteria, and submit your assignments by the assigned due date.

I've done the other formatting for [632](https://learn.twu.ca/course/view.php?id=30200&section=1#tabs-tree-start), but could you fix the books and the Week intros?  Thank you!

**Comment by Nataliadsanchez** [comment]
### **LDRS 632**
- [ ] There are no summaries
- [ ] There is no content in Unit 10, only Resources

**Broken Links**
- [ ] Resources in Unit 5: VIDEO trailer for Step One: Establishing a Sense of Urgency
- [ ] Resources in Unit 5: Everybody’s doing it. (Video 2:31)
- [ ] Learning Activity 4.1 in Unit 6: I couldn´t locate the resource because there is no name to it. It says: Watch: This link is dead.

**ASSESSMENT:**
- [ ] Expressive Change 3 - The Giving Field Forum: The link _The Giving Field_ is broken.
- [ ] Expressive Change 3 - The Giving Field Forum: There is not content when clicking to expand rubric

### **LDRS 631**
- [ ] Topic 5 in Unit 1 is very long. What about placing content into accordions?
- [ ] Topic 5 in Unit 1: The slide content at the end of it do not correspond to the description mentioned in the subtopic. Could you please locate this content? The topic is “The MA Leadership Competencies Framework” 
- [ ] Activity 5.1 in Unit 10: Please fix the formatting 

**Broken Links:**
- [ ] Last reference Unit 6. Zisa, J. (June 20, 2013). Listen to serve: Servant leadership and the practice of effective listening. Retrieved from: https://www.greenleaf.org/listen-to-serve-servant-leadership-and-the-practice-of-effective-listening/. 
- [ ] First reference in Unit 7: Anderson, G. (2014, May 21). The Servant Leader as Persuader. Retrieved from: https://the16percent.com/2014/05/21/the-servant-leader-as-persuader/. 
- [ ] Third Reference in Unit 8: Gilster, S. (2014). Service: A Model for Health Care Servant Leaders. Retrieved from https://www.greenleaf.org/s-e-r-v-i-c-e-a-model-for-health-care-servant-leaders/. 
- [ ] Last reference in Unit 9: Stallard, M.L. & Pankau, J. (n.d.). To boost productivity and innovation, fire up the people you lead. Retrieved from* https://www.greenleaf.org/winning-workplaces/workplace-resources/features/special-feature/to-boost-productivity-and-innovation-fire-up-the-people-you-lead/. 

**General note: If you are embedding the YouTube links, please turn off the Multimedia plugins.**

**Comment by kmarjanovic** [comment]
Thanks so much for your review, Natalia!
@MeeksonHundoo or @meher-ahmed could you check the items above - in particular the missing links?  I've often found them in the original FAR course.

Any requests/questions we need a SME for, we'll pass on to DeAnn.

**Comment by kmarjanovic** [comment]
Good morning, all!
In reviewing Moodle Books for [LDRS 620](https://learn.twu.ca/mod/book/view.php?id=932320&chapterid=10282) - I see there is additional numbering of Learning Activities that doesn't match the [MA Lead template](https://learn.twu.ca/mod/book/view.php?id=915827&chapterid=8640).  Also, the Checking Your Learning is supposed to be under the Summary section.  Is this the same in other courses?

e.g. MA Lead template
![image](https://github.com/TWUOnline/bug-tracking/assets/35148983/bfbd9852-8df8-4e52-96f9-38354e4cd42b)

compared to 620
![image](https://github.com/TWUOnline/bug-tracking/assets/35148983/290ec6a2-ff48-4fc8-aa3b-d73d0e986919)


I'll check the other courses to see if we need to fix this, but could someone fix it in 620?

**Comment by kmarjanovic** [comment]
Formatting also needs to be fixed in 501, 503, 591, 697, 617, 625, 632.

Since it seems to be in 8 of the courses, perhaps we should discuss first.  @cmadland, @Jeff-TWU , Is this automatic numbering?  Is it an easy fix?

These 2 courses also need formatting edits:

504 topics are missing numbers

612 is missing topic titles.  I see this is missing in Bookdown too - but it is there in the original [FAR course](https://learn.twu.ca/course/view.php?id=20674&section=0#tabs-tree-start).  Grav

**Comment by meher-ahmed** [comment]
Hi Kelly, I am checking on 620 and according to the [620 bookdown ](file:///Users/meher/Documents/GitHub/ldrs620/docs/worldview-formation-and-development.html#unit-1-summary)
after the summary section, we have an assessment, which is added separately in Moodle books. So the content will continue as summary, as check your learning and reff.  Some of the courses don't have summaries or other sections. Meekson and I have created Github issues on their respective course repos.

**Comment by Jeff-TWU** [comment]
> Formatting also needs to be fixed in 501, 503, 591, 697, 617, 625, 632.
> 
> Since it seems to be in 8 of the courses, perhaps we should discuss first. @cmadland, @Jeff-TWU , Is this automatic numbering? Is it an easy fix?
> 
> These 2 courses also need formatting edits:
> 
> 504 topics are missing numbers
> 
> 612 is missing topic titles. I see this is missing in Bookdown too - but it is there in the original [FAR course](https://learn.twu.ca/course/view.php?id=20674&section=0#tabs-tree-start). Grav

Auto numbering can be turned off in the course settings. 
![Screen Shot 2023-08-28 at 10 30 49 AM](https://github.com/TWUOnline/bug-tracking/assets/66963480/57fff7f5-b399-4125-815f-d8da84f8cab9)
Unit numbers would then have to be manually added in the title.

**Comment by meher-ahmed** [comment]
We have been using the numbering formatting in Bookdown for all chapters. Yes it is turned on in the settings and we are not individually numbering each units in 1 book

**Comment by kmarjanovic** [comment]
According to the template, the auto-numbering should be turned off, and the numbers inserted manually:

![image](https://github.com/TWUOnline/bug-tracking/assets/35148983/f8dc560e-2fcf-44c7-abd8-b7fe5ae312d7)

**Comment by kmarjanovic** [comment]
Good morning @chloecwh !  
I've been checking through the Working Masters & noticed on the Assessment page, the Assignment dropboxes are available but hidden.  They should all be showing for students.  (See MA Lead Master Template assessment section).

Unfortunately this means we need to check all MA Lead Working Masters assessment pages to see if the formatting is correct.  Could you or Meher check that today?

Also - could we confirm that all of the requests made above are complete?  There are certain items we need instructors to fix (broken links to websites/videos), but most other issues we can track down.  A common one is the pdf not being uploaded to Moodle. (original pdf in Grav course).

I'm just confirming with @deannadams which courses need to be ready for January, but I assume most of the following.  Deann, I would suggest the Working Master versions are the ones to import into the January shells, as hopefully instructors have made edits there.

[[LDRS 500: Leadership Foundations (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30331&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30331&section=0#tabs-tree-start)

[[LDRS 501: Strategic Leadership (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30332&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30332&section=0#tabs-tree-start)

[[LDRS 502: Team Leadership + Conflict Resolution (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30333&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30333&section=0#tabs-tree-start)

[[LDRS 503: Results-based Leadership (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30334&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30334&section=0#tabs-tree-start)

[[LDRS 504: Leadership Values & Ethics (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30335&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30335&section=0#tabs-tree-start)

[[LDRS 591: Scholarly Inquiry (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30336&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30336&section=0#tabs-tree-start)

[[LDRS 611: Organizational (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30337&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30337&section=0#tabs-tree-start)

[[LDRS 612: Leading Change & Innovation (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30338&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30338&section=0#tabs-tree-start)

[[LDRS 614: Mission, Marketing & Quality (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30339&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30339&section=0#tabs-tree-start)

[[LDRS 617: Employee Engagement & Stewardship (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30340&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30340&section=0#tabs-tree-start)

[[LDRS 620: Worldview Foundation Educ Leadership (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30345&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30345&section=0#tabs-tree-start)

[[LDRS 623: Developing Education Programs (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30346&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30346&section=0#tabs-tree-start)

[[LDRS 624: School Leadership & Supervision (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30351)](https://learn.twu.ca/course/view.php?id=30351)

[[LDRS 625: Educational Leadership & Dynamics (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30348&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30348&section=0#tabs-tree-start)

[[LDRS 631: Health Leadership Issues (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30343&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30343&section=0#tabs-tree-start)

[[LDRS 632: Leadership & Change (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30341&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30341&section=0#tabs-tree-start)

[[LDRS 634: Accountability & Performance (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30344&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30344&section=0#tabs-tree-start)

[[LDRS 697: Leadership Integration Project I (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30349&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30349&section=0#tabs-tree-start)

[[LDRS 698: Leadership Integration Project II (6 week Working Master)](https://learn.twu.ca/course/view.php?id=30350&section=0#tabs-tree-start)](https://learn.twu.ca/course/view.php?id=30350&section=0#tabs-tree-start)

**Comment by kmarjanovic** [comment]
Other edits:
LDRS 591 - [Assessment section](https://learn.twu.ca/course/view.php?id=30336&section=7#tabs-tree-start) has the wrong assessment.  See [FAR Master](https://learn.twu.ca/course/view.php?id=8924&section=14#tabs-tree-start) for assignment info

**Comment by meher-ahmed** [comment]
Hi @kmarjanovic for the assessment section do you want the current assessments and Discussion forum to be removed and add the same info from FAR Master in the section?

**Comment by chloecwh** [comment]
Thanks for hopping on the call! I will work on checking the assessment, update 591 and get it done today. Meher will start on confirming the requests. 
@kmarjanovic @meher-ahmed

**Comment by meher-ahmed** [comment]
Hi @cmadland, could you please let us know if we need to make the changes for the above courses in Bookdown or the Moodle shell? I will start working on the changes that are needed. Thank you.

**Comment by kmarjanovic** [comment]
@chloecwh & @meher-ahmed - here's an update of priority courses:

Course edits completed by instructors:  (Kelly will check these today)
1.	LDRS 612: Leading Change & Innovation (ID reviewing Assessment section)
2.	LDRS 617: Employee Engagement & Stewardship
3.	LDRS 623: Developing Education Programs
4.	LDRS 634: Accountability & Performance

Course edits due Dec. 14: (Production please prioritize these)
5.	LDRS 504: Leadership Values & Ethics
6.	LDRS 591: Scholarly Inquiry
7.	LDRS 611: Organizational Behaviour
8.	LDRS 620: Worldview Foundation Educ Leadership
9.	LDRS 624: School Leadership & Supervision
10.	LDRS 632: Leadership & Change 
11.	LDRS 697: Leadership Integration Project I
12.	LDRS 698: Leadership Integration Project II

Course edits due Feb. 29:
13.	LDRS 503: Results-based Leadership

Due dates for the following TBD
14.	LDRS 500: Leadership Foundations 
15.	LDRS 501: Strategic Leadership
16.	LDRS 502: Team Leadership + Conflict Resolution
17.	LDRS 614: Mission, Marketing & Quality
18.	LDRS 631: Health Leadership Issues
19.	LDRS 625: Educational Leadership & Dynamics

**Comment by chloecwh** [comment]
@kmarjanovic 
1. updated - Assignment dropboxes are available but hidden
2. updated - LDRS 591 assessment with APA guidline

**Comment by meher-ahmed** [comment]
Hi @kmarjanovic for [LDRS 591 assignment Article Critique](https://learn.twu.ca/course/modedit.php?update=967712&return=0&sr=7)s in the grave site there is an attachment [assignment 3](https://learn.twu.ca/course/view.php?id=8924&section=18#tabs-tree-start) at the bottom of the section, where in Moodle books do you want that to go? do you want this to be somewhere attached within the assignment as a line and add the file or separately add the file as a page in the assessment section?

**Comment by meher-ahmed** [comment]
@kmarjanovic [LDRS 591 ](https://learn.twu.ca/course/view.php?id=30336&section=7) Assessment page has been update with correct assignment and discussion forums. Please review for edits.

**Comment by kmarjanovic** [comment]
Thanks so much, @meher-ahmed !
Just a couple of edits:
- [ ] Please show all discussions in the assessment page

- [ ] For Rubric for Assignment # 1- it's difficult to read.  Would making the table lines visible help?  Let's test out some options.

- [ ] Literature Search Assignment Step 4 - please take out the video embed.  It should be "Here is a helpful video explaining [How to Read an Academic Paper](https://youtu.be/SKxm2HF_-k0)."
- [ ] Step 4 - Table 1. Literature Review Search Summary Matrix in same assignment - the table is not clear, as it is in the original grav.  Show lines perhaps?
- [ ] Step 6 - fix formatting.  original had boxes - but we can have an indent or box to set it apart.
- [ ] Rubric for the [Literature Search](https://learn.twu.ca/mod/assign/view.php?id=967686) could be more clear with table lines showing.


- [ ] Assignment: Scholarly Article Review - can take out this title in text box (it's not in other assignments)
- [ ] fix formatting (3 vertical lines should be 1) & rubrics


- [ ] Article Critiques - fix formatting para 1
- [ ] fix formatting and rubric

- [ ] APA Formatting section.  Formatting of table needs to be fixed.   See original course.
- [ ] 
![image](https://github.com/TWUOnline/bug-tracking/assets/35148983/4e98ce12-c7f9-40e3-b46e-f9930e0298e2)

- [ ] Please bold the text that is bold in the original course.
- [ ] Please check bullets for In-text Citations section as some are combined, others are missing sub-bullets, some points have a bullet when they should not, etc.

**Comment by kmarjanovic** [comment]
@chloecwh in the Weeks sections - the Assignment section should not be hidden.  Otherwise students will not know what is due.  Please see the [MA Lead Template](https://learn.twu.ca/course/view.php?id=30164&section=2#tabs-tree-start)

e.g. We want to follow this - the order - heading levels, bullets, etc.  Articles are hidden.  Dropboxes are shown only in the assessment section of the course (not in weeks sections).

## Welcome to Week 2 of LDRS #!
On your own: Work through the content and learning activities in Units 2 & 3.
Meet with your class. Be sure to check the [announcements](https://learn.twu.ca/mod/forum/view.php?id=915814) for specific required activities and resources.

See below for the course materials you will complete this week, which include learning outcomes, learning activities, and resources. Be sure to review the assessment instructions, including grading criteria, and submit your assignments by the assigned due date.

###Assessment
- Forum: [Learning Reflection #2](https://learn.twu.ca/mod/forum/view.php?id=915834)
- [“Empowering others on the team” individual notes](https://learn.twu.ca/mod/assign/view.php?id=915835)

[Formative “Team Toolkit” - Group](https://learn.twu.ca/mod/assign/view.php?id=915836)

###Week 2 Resources
book icon 
[Unit 2 Empowering Others on a Team](https://learn.twu.ca/mod/book/view.php?id=915832)

**Comment by kmarjanovic** [comment]
@chloecwh & @meher-ahmed 

Please see the MA Lead Template for how to list and set up the Assignments in the MA Lead courses.

[LDRS 591](https://learn.twu.ca/course/view.php?id=30336&section=1#tabs-tree-start) has 2 assignment dropboxes per assignment - which is incorrect. 

The assignment dropboxes & discussion forums are only in the main Assessment section.  They are all available and show on the page.

In the Weeks sections - we have a bulleted list that includes the link to the assignment dropboxes and discussion forums.  

Please let me know if you have any questions about this.

**Comment by meher-ahmed** [comment]
@kmarjanovic, the 3 documents attached in [591: Scholarly Inquiry (FAR Master)](https://learn.twu.ca/course/view.php?id=8924&section=15#tabs-tree-start) summary of Participation/Group Work section has no mention within the content of the section that I can embed it so can I just create a new line within the page and embed the documents or would you want each file to be added separately outside the page?

**Comment by meher-ahmed** [comment]
Hi @kmarjanovic [LDRS 591: Scholarly Inquiry (6 week Working Master)](urhttps://learn.twu.ca/course/view.php?id=30336&section=7l) is -

- [ ] updated from grav to bookdown and then to Moodle Books. 
- [ ] The edits have been made as requested
- [ ] New tables for each assignment have been created separately for Moodle Books as from Bookdown, when I copy the tables, it does not render properly.
- [ ] For each assignment, template files have been added within the assignment as embedded links. In the [grave course](https://learn.twu.ca/course/view.php?id=8924&section=15#tabs-tree-start), the files are attached separately. (Pls let me know if that is how you need it or if you would want these sources to be added outside the section).
- [ ] For the Participation/Group Work page I have added all the 3 sources at the bottom of the page but if you want me to move them somewhere above the rubric pls letme know. 
<img width="1440" alt="Screenshot 2023-12-14 at 11 04 01 AM" src="https://github.com/TWUOnline/bug-tracking/assets/130523647/a0e59e1a-dd5b-4708-8ef1-1f40ff431c7c">
Please have a look and letme know on any edits or changes. Thank you!1

**Comment by meher-ahmed** [comment]
<img width="1343" alt="Screenshot 2023-12-14 at 11 08 44 AM" src="https://github.com/TWUOnline/bug-tracking/assets/130523647/6a12f2d8-6946-4006-aba1-e9ac264ba17b">
 sorry this is the correct text

**Comment by kmarjanovic** [comment]
This is great, Meher!
Just for those last 3 resources, let's put the link in the specific related activity:
Unit 2 Group Activity. "Resource: Use Evaluating Resources: the CRAAP TEST" 

The APA Presentation (Exemplar).pdf goes under Unit 3 Group Activity. (Link: Resource: APA Presentation (Exemplar))

The Action Research Presentation (Exemplar).pdf goes under Unit 8 Group Activity. (Link: Resource: Action Research Presentation (Exemplar))

**Comment by meher-ahmed** [comment]
Hi @kmarjanovic its done please have a look https://learn.twu.ca/mod/page/view.php?id=1002739&forceview=1

**Comment by kmarjanovic** [comment]
Great!  Thank you!
Could you also Show all the discussions on the Assessment page?
Also Discussions title should be Heading level 3 (& make i lower case in Discussions title)

**Comment by chloecwh** [comment]
Hi @kmarjanovic , I've checked all the formatting for the assessment and discussion forums of:
LDRS 612: Leading Change & Innovation (ID reviewing Assessment section)
LDRS 617: Employee Engagement & Stewardship
LDRS 623: Developing Education Programs
LDRS 634: Accountability & Performance
Course edits due Dec. 14: (Production please prioritize these)
5. LDRS 504: Leadership Values & Ethics
6. LDRS 591: Scholarly Inquiry
7. LDRS 611: Organizational Behaviour
8. LDRS 620: Worldview Foundation Educ Leadership
9. LDRS 624: School Leadership & Supervision
10. LDRS 632: Leadership & Change
11. LDRS 697: Leadership Integration Project I
12. LDRS 698: Leadership Integration Project II

[**591**](https://learn.twu.ca/course/view.php?id=30336&section=5#tabs-tree-start) 

- I noticed the discussion forums of naming is different than the mentioned assignment in each units. 

**612**
- assignment [Expressive Change: Phase 1 Introduction Day 1](https://learn.twu.ca/mod/assign/view.php?id=967803) this is a dropbox, can I make it a text area instead? 

- [assessment page](https://learn.twu.ca/course/view.php?id=30338&section=7#tabs-tree-start) is hard to follow. we can clean up the description by hiding it, so the description only stays inside the dropbox

**Comment by kmarjanovic** [comment]
I've looked through this page - and it seems all issues are addressed, or SMEs will fix in the working masters.

Thank you @meher-ahmed @Jeff-TWU @chloecwh @cmadland for all your work on these courses!!
